src/utils/achievements/categoryCompletionAchievement.ts

```diff
@@ -1,4 +1,6 @@
 import { of } from "await-of";
+import { cpuUsage } from "process";
```

Comment:
What is this ?

Reply:
What the? 🤔  Didn't see it, I will remove it.

Reply:
Autocompletion misfire, probably :)